File: PC_Python/PC2_Calculadora_IMC_diagnóstico.py
peso = float(input("Ingrese su peso Kg: "))
altura = float(input("Ingrese su talla en m: "))

if peso >= 5 and peso <= 400 and altura >= 0.6 and altura <= 2.5:
    
    imc = peso / (altura ** 2)
    
    print("Su IMC es:", imc)

    if imc < 18.5:
        print("Bajo peso")
    elif imc < 25:
        print("Peso normal")
    elif imc < 30:
        print("Sobrepeso")
    else:
        print("Obesidad")
else:
    print("Datos de entrada inválidos.")

File: PC_C_Char/PC2_Calculadora_IMC_diagnóstico.cs
Console.Write("Ingrese su peso Kg: ");
        double peso = double.Parse(Console.ReadLine());
        Console.Write("Ingrese su talla en m: ");
        double altura = double.Parse(Console.ReadLine());

        if (peso >= 5 && peso <= 400 && altura >= 0.6 && altura <= 2.5)
        {
            double imc = peso / Math.Pow(altura, 2);

            Console.WriteLine("Su IMC es: " + imc);
            
            if (imc < 18.5) 
                Console.WriteLine("Bajo peso");
            else if (imc < 25) 
                Console.WriteLine("Peso normal");
            else if (imc < 30) 
                Console.WriteLine("Sobrepeso");
            else 
                Console.WriteLine("Obesidad");
        }
        else
        {
            Console.WriteLine("Datos de entrada inválidos.");
        }